blocks/init/src/Blocks/components/video/video.php

```diff
@@ -65,7 +65,6 @@
 		<iframe
 		class="<?php echo \esc_attr($videoClass); ?>"
 		src="<?php echo \esc_url($localUrl); ?>"
-		frameBorder="0"
```

Comment:
Isn't this some compatibility thing / reset for older browsers (IE11?)

Reply:
phpstorm said it's deprecated, and I don't think we should bother with IE11 support (even MS killed it 🤷🏼‍♂️ )